packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -606,6 +636,19 @@ class _RailDestination extends StatelessWidget {
         final double verticalPadding = lerpDouble(_verticalDestinationPaddingNoLabel, _verticalDestinationPaddingWithLabel, appearingAnimationValue)!;
         final Interval interval = selected ? const Interval(0.25, 0.75) : const Interval(0.75, 1.0);
         final Animation<double> labelFadeAnimation = destinationAnimation.drive(CurveTween(curve: interval));
+
+        final Widget iconWithIndicator =  Stack(
+          alignment: Alignment.center,
+          children: <Widget>[
+            if (useIndicator)
+              NavigationIndicator(
+                animation: destinationAnimation,
+                width: 56,
+                borderRadius: 100,
```

Comment:
Same nit as above, the 100 border radius feels weird to me

Reply:
Corrected value.